docs/community/com-config/com-config-manage-fb-services.md

```diff
@@ -16,10 +16,11 @@ You can start, stop and check the status of FeatureBase services using `systemct
 
 {% include /com-install/com-install-before-begin.md %}
 * Obtain `sudo` credentials for the system
+* [Setup FeatureBase as a service](/docs/community/old-setup/old-community-quickstart-guide/set-up-log-and-data-folders)
 
 ## Manage FeatureBase services
 
-* Open a CLI and CD to the `/featurebase` installation directory.
+* Open a CLI and the `/featurebase` installation directory.
```

Comment:
CD is fine, trying to avoid confusion that /featurebase is in the root. */featurebase